assembly/src/ast/nodes/serde/debug.rs

```diff
@@ -0,0 +1,32 @@
+use super::{super::DebugOptions, ByteReader, ByteWriter, DeserializationError, ToString};
+
+const STACK_ALL: u8 = 0;
```

Comment:
Is `0` and `1` correct here?

Reply:
Yes, these are fine. These can be viewed as enum values for `DebugOption` variants (we can't assign these values directly as enum variants are complex).